tests/integrationv2/README.md

```diff
@@ -1,26 +1,39 @@
 # Quick start
 
-You need to install Tox to help setup your environment. Once tox is installed, run it to setup your
-local environment. Also set your path to access the tox artifacts.
+You must have run through the standard codebuild setup as described in the root README. This will make sure you
+have all the dependencies installed correctly. The integration test dependencies are:
 
-```
-$ tox
-py38 create: /home/ubuntu/development/s2n/tests/integrationv2/.tox/py38
-py38 installdeps: pep8, pytest==5.3.5, pytest-xdist
-py38 installed: apipkg==1.5,attrs==19.3.0,execnet==1.7.1,importlib-metadata==1.6.0,more-itertools==8.2.0,packaging==20.3,pep8==1.7.1,pkg-resources==0.0.0,pluggy==0.13.1,py==1.8.1,pyparsing==2.4.7,pytest==5.3.5,pytest-forked==1.1.3,pytest-xdist==1.32.0,six==1.14.0,wcwidth==0.1.9,zipp==3.1.0
+ * s2nc and s2nd (should be in the bin/ directory)
+ * libs2n (should be in the lib/ directory)
+ * openssl (based on the S2N_LIBCRYPTO env var)
+ * tox
```

Comment:
We could add a note about pytest, that it should be from `.tox/py38/bin/pytest`

Reply:
I left pytest out because this change removes the need for it. The previous instructions set the PATH variable, but this is fragile. So removing the pytest step altogether will be better long term.